diff --git a/src/AccountId.php b/src/AccountId.php
index dc062df..519290b 100644
--- a/src/AccountId.php
+++ b/src/AccountId.php
@@ -1,8 +1,4 @@
 <?php
-/**
- * @file
- * Contains \Drupal\optimizely\AccountId
- */
 
 namespace Drupal\optimizely;
 
diff --git a/src/AccountSettingsForm.php b/src/AccountSettingsForm.php
index f9a0dea..473b731 100644
--- a/src/AccountSettingsForm.php
+++ b/src/AccountSettingsForm.php
@@ -1,10 +1,5 @@
 <?php
 
-/**
- * @file
- * Contains \Drupal\optimizely\AccountInfoForm
- */
-
 namespace Drupal\optimizely;
 use Drupal\Core\Form\FormBase;
 use Drupal\Core\Form\FormStateInterface;
diff --git a/src/AddUpdateForm.php b/src/AddUpdateForm.php
index 215acbd..06a9996 100644
--- a/src/AddUpdateForm.php
+++ b/src/AddUpdateForm.php
@@ -1,10 +1,5 @@
 <?php
 
-/**
- * @file
- * Contains \Drupal\optimizely\AddUpdateForm
- */
-
 namespace Drupal\optimizely;
 
 use Drupal\Core\Form\FormBase;
diff --git a/src/AjaxEnable.php b/src/AjaxEnable.php
index 72bc2a8..29273c4 100644
--- a/src/AjaxEnable.php
+++ b/src/AjaxEnable.php
@@ -1,10 +1,5 @@
 <?php
 
-/**
- * @file
- * Contains \Drupal\optimizely\src\AjaxEnable
- */
-
 namespace Drupal\optimizely;
 
 use Symfony\Component\HttpFoundation\JsonResponse;
diff --git a/src/DeleteForm.php b/src/DeleteForm.php
index 0269fdc..66ab8ab 100644
--- a/src/DeleteForm.php
+++ b/src/DeleteForm.php
@@ -1,10 +1,5 @@
 <?php
 
-/**
- * @file
- * Contains \Drupal\optimizely\DeleteForm
- */
-
 namespace Drupal\optimizely;
 
 use Drupal\Core\Form\ConfirmFormBase;
diff --git a/src/LookupPath.php b/src/LookupPath.php
index e9a00fd..149a9df 100644
--- a/src/LookupPath.php
+++ b/src/LookupPath.php
@@ -1,10 +1,5 @@
 <?php
 
-/**
- * @file
- * Contains \Drupal\optimizely\LookupPath
- */
-
 namespace Drupal\optimizely;
 
 /**
diff --git a/src/PathChecker.php b/src/PathChecker.php
index e015129..faed7b7 100644
--- a/src/PathChecker.php
+++ b/src/PathChecker.php
@@ -1,10 +1,5 @@
 <?php
 
-/**
- * @file
- * Contains \Drupal\optimizely\src\PathChecker
- */
-
 namespace Drupal\optimizely;
 
 
diff --git a/src/ProjectListForm.php b/src/ProjectListForm.php
index 2f51382..88af74f 100644
--- a/src/ProjectListForm.php
+++ b/src/ProjectListForm.php
@@ -1,10 +1,5 @@
 <?php
 
-/**
- * @file
- * Contains \Drupal\optimizely\ProjectListForm
- */
-
 namespace Drupal\optimizely;
 
 use Drupal\Core\Form\FormBase;
diff --git a/src/Tests/OptimizelyAccessTest.php b/src/Tests/OptimizelyAccessTest.php
index 57d1205..f7624d0 100644
--- a/src/Tests/OptimizelyAccessTest.php
+++ b/src/Tests/OptimizelyAccessTest.php
@@ -1,10 +1,5 @@
 <?php
 
-/**
- * @file
- * Contains \Drupal\optimizely\Tests\OptimizelyAccessTest
- */
-
 namespace Drupal\optimizely\Tests;
 
 use Drupal\simpletest\WebTestBase;
diff --git a/src/Tests/OptimizelyAddUpdateTest.php b/src/Tests/OptimizelyAddUpdateTest.php
index 75b0ad4..237413b 100644
--- a/src/Tests/OptimizelyAddUpdateTest.php
+++ b/src/Tests/OptimizelyAddUpdateTest.php
@@ -1,10 +1,5 @@
 <?php
 
-/**
- * @file
- * Contains \Drupal\optimizely\Tests\OptimizelyAddUpdateTest
- */
-
 namespace Drupal\optimizely\Tests;
 
 use Drupal\simpletest\WebTestBase;
diff --git a/src/Tests/OptimizelyAliasIncludeSnippetTest.php b/src/Tests/OptimizelyAliasIncludeSnippetTest.php
index a6d65dd..86c55dc 100644
--- a/src/Tests/OptimizelyAliasIncludeSnippetTest.php
+++ b/src/Tests/OptimizelyAliasIncludeSnippetTest.php
@@ -1,10 +1,5 @@
 <?php
 
-/**
- * @file
- * Contains \Drupal\optimizely\Tests\OptimizelyAliasIncludeSnippetTest
- */
-
 namespace Drupal\optimizely\Tests;
 
 use Drupal\simpletest\WebTestBase;
diff --git a/src/Tests/OptimizelyDefaultProjectTest.php b/src/Tests/OptimizelyDefaultProjectTest.php
index af8b27e..7da465b 100644
--- a/src/Tests/OptimizelyDefaultProjectTest.php
+++ b/src/Tests/OptimizelyDefaultProjectTest.php
@@ -1,10 +1,5 @@
 <?php
 
-/**
- * @file
- * Contains \Drupal\optimizely\src\Tests\OptimizelyDefaultProjectTest
- */
-
 namespace Drupal\optimizely\Tests;
 
 use Drupal\simpletest\WebTestBase;
diff --git a/src/Tests/OptimizelyDefaultSettingsTest.php b/src/Tests/OptimizelyDefaultSettingsTest.php
index d4f3057..c8c9713 100644
--- a/src/Tests/OptimizelyDefaultSettingsTest.php
+++ b/src/Tests/OptimizelyDefaultSettingsTest.php
@@ -1,10 +1,5 @@
 <?php
 
-/**
- * @file
- * Contains \Drupal\optimizely\src\Tests\OptimizelyDefaultSettingsTest
- */
-
 namespace Drupal\optimizely\Tests;
 
 use Drupal\simpletest\WebTestBase;
diff --git a/src/Tests/OptimizelyEnableDisableTest.php b/src/Tests/OptimizelyEnableDisableTest.php
index 0cfb54e..18010e8 100644
--- a/src/Tests/OptimizelyEnableDisableTest.php
+++ b/src/Tests/OptimizelyEnableDisableTest.php
@@ -1,10 +1,5 @@
 <?php
 
-/**
- * @file
- * Contains \Drupal\optimizely\Tests\OptimizelyEnableDisableTest
- */
-
 namespace Drupal\optimizely\Tests;
 
 use Drupal\simpletest\WebTestBase;
diff --git a/src/Tests/OptimizelyIncludeSnippetTest.php b/src/Tests/OptimizelyIncludeSnippetTest.php
index 2c86496..6152dba 100644
--- a/src/Tests/OptimizelyIncludeSnippetTest.php
+++ b/src/Tests/OptimizelyIncludeSnippetTest.php
@@ -1,10 +1,5 @@
 <?php
 
-/**
- * @file
- * Contains \Drupal\optimizely\Tests\OptimizelyIncludeSnippetTest
- */
-
 namespace Drupal\optimizely\Tests;
 
 use Drupal\simpletest\WebTestBase;
diff --git a/src/Tests/OptimizelyPageSnippetTest.php b/src/Tests/OptimizelyPageSnippetTest.php
index a3e4b9c..92c4636 100644
--- a/src/Tests/OptimizelyPageSnippetTest.php
+++ b/src/Tests/OptimizelyPageSnippetTest.php
@@ -1,10 +1,5 @@
 <?php
 
-/*
- * @file
- * Contains \Drupal\optimizely\Tests\OptimizelyPageSnippetTest
- */
-
 namespace Drupal\optimizely\Tests;
 
 use Drupal\simpletest\WebTestBase;
diff --git a/src/Tests/OptimizelySchemaTest.php b/src/Tests/OptimizelySchemaTest.php
index 32a1cd8..c88b714 100644
--- a/src/Tests/OptimizelySchemaTest.php
+++ b/src/Tests/OptimizelySchemaTest.php
@@ -1,10 +1,5 @@
 <?php
 
-/**
- * @file
- * Contains \Drupal\optimizely\Tests\OptimizelySchemaTest
- */
-
 namespace Drupal\optimizely\Tests;
 
 use Drupal\simpletest\WebTestBase;
